Formula/ungit.rb

```diff
@@ -25,7 +25,7 @@ def install
     require "nokogiri"
 
     pid = fork do
-      exec bin/"ungit", "--no-launchBrowser", "--autoShutdownTimeout", "5000" # give it an idle timeout to make it exit
+      exec bin/"ungit", "--no-launchBrowser", "--autoShutdownTimeout", "7000" # give it an idle timeout to make it exit
     end
     sleep 5
```

Comment:
I'm not familiar with ruby but why `sleep 5`? It better tell to parent process to wait for the child to exit like:
```rb
_, status = Process.waitpid2(pid)
```

Reply:
But we‘re waiting here to give the server time to start.

If we‘re waiting here until the server shuts down, the test will always fail, because the curl from the line below will never reach the server. We just want to ensure that the process is always killed after the curl request succeeded.